content/en/observability_pipelines/production_deployment_overview/_index.md

```diff
@@ -1,42 +1,188 @@
+
+Alternatively, you can deploy the Observability Pipelines Worker on separate nodes as an aggregator. See [Choosing where to process data](#choosing-where-to-process-data) for more details.
+
+##### Reducing agent risk
+
+When integrating with an agent, configure the agent to be a simple data forwarder and route supported data types through the Observability Pipelines Worker. This reduces the risk of data loss and service disruption by minimizing the agent's responsibilities.
+
+## Processing data
+
+If you want to design an efficient pipeline between your Observability Pipelines Worker's sources and sinks, it helps to understand which types of data to process and where to process it.
+
+### Choosing which data to process
+
+You can use Observability Pipelines Worker to process logs, metrics, and traces. However, real-time, vendor-specific data, such as continuous profiling data, is not interoperable and typically does not benefit from processing.
+
+### Choosing where to process data
+
+Observability Pipelines Worker can be deployed anywhere in your infrastructure. Deploy it directly on your node as an agent for local processing, or on separate nodes as an aggregator for remote processing. Where the processing happens depends largely on your use case and environment.
+
+#### Local processing
+
+With local processing, Observability Pipelines Worker is deployed on each node as an agent. 
+
+{{< img src="observability_pipelines/production_deployment_overview/agent.png" alt="A diagram showing two separate nodes, each containing services, other agents, other node data, and an Observability Pipelines Worker, and the node and agent data is sent to the Worker, and then to the sinks" style="width:70%;" >}}
+
+Data is processed on the same node from which the data originated. This provides operational simplicity since the Observability Pipelines Worker has direct access to your data and scales along with your infrastructure.
+
+Local processing is recommended for:
+
+- Simple environments that do not require high durability or high availability.
+- Use cases, such as fast, stateless processing, and streaming delivery, that do not require holding onto data for long periods of time.
+- Operators that can make node-level changes without a lot of friction.
+
+#### Remote processing
+
+For remote processing, the Observability Pipelines Worker can be deployed on separate nodes as an aggregator. 
+
+{{< img src="observability_pipelines/production_deployment_overview/aggregator_role.png" alt="A diagram showing an Observability Pipelines Worker aggregator containing multiple Workers that are receiving data from the network load balancer and sending data to different sinks" style="width:100%;" >}}
```

Comment:
Could you have the brand design team adjust the drawing to remove the two green "OPWs" from "Sources"?

Reply:
cc/ @drichards-87

Reply:
Just submitted the request to Brand Design. I'll let you know when I hear back from them.